packages/components/src/Form/src/Form.tsx

```diff
@@ -0,0 +1,102 @@
+import {
+    ButtonContext,
+    TextFieldContext
+} from "@hopper-ui/components";
+import {
+    useResponsiveValue,
+    useStyledSystem,
+    type StyledComponentProps,
+    type ResponsiveProp
+} from "@hopper-ui/styled-system";
+import clsx from "clsx";
+import { forwardRef, type ForwardedRef, type CSSProperties } from "react";
+import { useContextProps, Form as RACForm, type FormProps as RACFormProps } from "react-aria-components";
+
+import { cssModule, SlotProvider } from "../../utils/index.ts";
+
+import { FormContext } from "./FormContext.ts";
+
+import styles from "./Form.module.css";
+
+export const GlobalFormCssSelector = "hop-Form";
+
+export interface FormProps extends StyledComponentProps<RACFormProps> {
+    /**
+     * Whether the form elements are disabled.
+     */
+    isDisabled?: boolean;
+
+    /**
+     * If `true`, the TextField will take all available width.
+     */
+    isFluid?: ResponsiveProp<boolean>;
+
+    /**
+     * The size of the TextField.
```

Comment:
Shouldn't this be the size of the Form? not TextField

Reply:
alright so from what I can see, the size's only purpose it to be passed on to the fields.
Maybe you could say something like:
The size of the fields within the form.